sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/api/Microsoft.Azure.WebJobs.Extensions.SignalRService.netstandard2.0.cs

```diff
@@ -104,23 +106,23 @@ public abstract partial class ServerlessHub : System.IDisposable
         protected virtual void Dispose(bool disposing) { }
         protected System.Collections.Generic.IList<System.Security.Claims.Claim> GetClaims(string jwt) { throw null; }
         protected Microsoft.Azure.WebJobs.Extensions.SignalRService.SignalRConnectionInfo Negotiate(string userId = null, System.Collections.Generic.IList<System.Security.Claims.Claim> claims = null, System.TimeSpan? lifetime = default(System.TimeSpan?)) { throw null; }
-        protected System.Threading.Tasks.Task<Microsoft.Azure.WebJobs.Extensions.SignalRService.SignalRConnectionInfo> NegotiateAsync(Microsoft.Azure.SignalR.Management.NegotiationOptions options) { throw null; }
+        protected virtual System.Threading.Tasks.Task<Microsoft.Azure.WebJobs.Extensions.SignalRService.SignalRConnectionInfo> NegotiateAsync(Microsoft.Azure.SignalR.Management.NegotiationOptions options) { throw null; }
         [System.AttributeUsageAttribute(System.AttributeTargets.Class)]
         protected internal partial class SignalRConnectionAttribute : System.Attribute, Microsoft.Azure.WebJobs.IConnectionProvider
         {
             public SignalRConnectionAttribute(string connectionStringSetting) { }
             public string Connection { get { throw null; } set { } }
         }
     }
-    public abstract partial class ServerlessHub<T> where T : class
+    public abstract partial class ServerlessHub<T> : Microsoft.Azure.WebJobs.Extensions.SignalRService.ServerlessHub where T : class
     {
-        protected ServerlessHub(Microsoft.Azure.SignalR.Management.ServiceHubContext<T> serviceHubContext = null) { }
-        public Microsoft.Azure.SignalR.Management.ClientManager ClientManager { get { throw null; } }
-        public Microsoft.AspNetCore.SignalR.IHubClients<T> Clients { get { throw null; } }
-        public Microsoft.Azure.SignalR.Management.GroupManager Groups { get { throw null; } }
-        public Microsoft.Azure.SignalR.Management.UserGroupManager UserGroups { get { throw null; } }
-        protected static System.Collections.Generic.IList<System.Security.Claims.Claim> GetClaims(string jwt) { throw null; }
-        protected System.Threading.Tasks.Task<Microsoft.Azure.WebJobs.Extensions.SignalRService.SignalRConnectionInfo> NegotiateAsync(Microsoft.Azure.SignalR.Management.NegotiationOptions options) { throw null; }
+        protected ServerlessHub() : base (default(Microsoft.Azure.SignalR.Management.IServiceHubContext), default(Microsoft.Azure.SignalR.Management.IServiceManager)) { }
+        protected ServerlessHub(Microsoft.Azure.SignalR.Management.ServiceHubContext<T> serviceHubContext) : base (default(Microsoft.Azure.SignalR.Management.IServiceHubContext), default(Microsoft.Azure.SignalR.Management.IServiceManager)) { }
+        public override Microsoft.Azure.SignalR.Management.ClientManager ClientManager { get { throw null; } }
```

Comment:
Does it make sense that this one overrides where the other properties hide? If someone has a `ServerlessHub<T>` typed as a `ServerlessHub` will the behavior be consistent between the different properties?